browser/static/js/main_zstack.js

```diff
@@ -732,9 +747,11 @@ let mousedown = false;
 let spacedown = false;
 var tooltype = 'draw';
 var project_id;
-var brush;
 let mouse_trace = [];
 
+var brush;
+var adjust;
```

Comment:
If these are only ever assigned once, we can set them to `const` instead of `var`.

Reply:
Currently this is still tricky to do. The `ImageAdjuster` class needs to be initialized with height and width attributes, which we don't know until information about the specific file is loaded. So that gets initialized in the `start_caliban` function. But the `adjuster` object still needs to have global scope (for now) to play nice with everything else, so that variable gets declared outside of `start_caliban`, which won't work with `const`. So I'm not sure we can easily make that switch until other big pieces of the code get refactored as well.